tests/data_context/migrator/test_cloud_migrator.py

```diff
@@ -8,95 +8,43 @@
 import great_expectations.exceptions as ge_exceptions
 from great_expectations import CloudMigrator
 from great_expectations.core.usage_statistics.events import UsageStatsEvents
+from great_expectations.data_context.migrator.cloud_migrator import MigrationResponse
 from great_expectations.data_context.store.ge_cloud_store_backend import (
     GeCloudRESTResource,
 )
+from tests.data_context.migrator.conftest import StubBaseDataContext
 
 
-@pytest.mark.unit
-@pytest.mark.cloud
-def test__send_configuration_bundle_sends_valid_http_request(
-    serialized_configuration_bundle: dict,
+@pytest.fixture
+def migrator_factory(
```

Comment:
Same note here - with both these factories, we have the ability to test a variety of configs